server/game/core/ability/abilityTargets/CardTargetResolver.ts

```diff
@@ -45,38 +40,29 @@ class CardTargetResolver {
         return CardSelector.for(Object.assign({}, properties, { cardCondition: cardCondition, targets: true }));
     }
 
-    getContextCopy(card, context) {
-        let contextCopy = context.copy();
+    private getContextCopy(card: Card, context: AbilityContext) {
+        const contextCopy = context.copy({});
```

Comment:
Let's just add a default value to the `copy()` method so we don't have to do this